ui-packages/packages/trusty/src/components/Organisms/CounterfactualOutcomeSelection/CounterfactualOutcomeSelection.tsx

```diff
@@ -79,82 +90,102 @@ const CounterfactualOutcomeSelection = (
   return (
     <>
       <Modal
-        variant={ModalVariant.medium}
+        variant={ModalVariant.large}
         aria-label="Counterfactual desired outcomes"
-        title="Select the desired outcomes"
+        title="Specify desired outcomes"
         isOpen={isOpen}
         onClose={onClose}
-        description="Define the desired outcome that must be achieved by the counterfactual scenarios."
+        description="Specify desired counterfactual outcomes for one or more original decision outcomes."
         actions={[
-          <Button
-            key="confirm"
-            variant="primary"
-            onClick={handleApply}
-            isDisabled={!isDesiredOutcomeDefined()}
+          // See https://github.com/patternfly/patternfly-react/pull/5991.
+          // We will currently get a memory leak error. Fixed in @patternfly/react-core@4.135.6
+          <Tooltip
```

Comment:
just wondering, should the tool-tip be displayed only when the button is disabled? do we need it when everything is O.K.?

Reply:
No we don't. I'll make the change :+1: